site/public/scss/_header.scss

```diff
@@ -88,6 +88,25 @@ header {
     }
 }
 
+.header__center__github {
+    position: absolute;
+    top: 10px;
+    bottom: 0;
+    left: 50%;
+
+    width: 40px;
+    height: 40px;
```

Comment:
isn't this automatically set? or no?
If not, let's calculate it using $header-height - 2*$header-padding